### DISCUSSION TITLE
How to install for User level only - Mac?

### DISCUSSION THREAD

**Discussion by TheBeardedBerserkr** [starship/starship#4911] [General]
Hey so my company has policies in place to not allow root access on our work laptops, how do run this for my user level only?

**Comment by chipbuster** [comment]
If you have access to homebrew, I believe we should be available there.

If not, you can use the install script with the -b option to specify the target directory, e.g.

```
curl -sS https://starship.rs/install.sh
sh install.sh -b ~my/program/directory
rm install.sh
```

**Reply by mickimnet** [reply]
> If you have access to homebrew, I believe we should be available there.

Yes, we are: `brew install starship` works.

**Reply by TheBeardedBerserkr** [reply]
So we don't have access to brew, that would make too much sense.
Installed the package via
```sh install.sh -b ~my/program/directory```
but also needed to add to $PATH
```export PATH="~my/program/directory";$PATH```
